v19.2/create-table.md

```diff
@@ -103,140 +105,132 @@ In CockroachDB, every table requires a [primary key](primary-key.html). If one i
-| logoff_date | DATE      |    true     | NULL           |                       | {}                                  |
-+-------------+-----------+-------------+----------------+-----------------------+-------------------------------------+
-(3 rows)
+  column_name | data_type | is_nullable | column_default | generation_expression |                indices                | is_hidden
++-------------+-----------+-------------+----------------+-----------------------+---------------------------------------+-----------+
+  id          | UUID      |    false    | NULL           |                       | {primary,users_name_key,users_dl_key} |   false
+  city        | STRING    |    true     | NULL           |                       | {}                                    |   false
+  name        | STRING    |    true     | NULL           |                       | {users_name_key}                      |   false
+  address     | STRING    |    true     | NULL           |                       | {}                                    |   false
+  credit_card | STRING    |    true     | NULL           |                       | {}                                    |   false
+  dl          | STRING    |    true     | NULL           |                       | {users_dl_key}                        |   false
+(6 rows)
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW INDEX FROM logoff;
+> SHOW INDEX FROM users;
 ~~~
 
 ~~~
-+------------+-----------------------+------------+--------------+-------------+-----------+---------+----------+
-| table_name |      index_name       | non_unique | seq_in_index | column_name | direction | storing | implicit |
-+------------+-----------------------+------------+--------------+-------------+-----------+---------+----------+
-| logoff     | primary               |   false    |            1 | user_id     | ASC       |  false  |  false   |
-| logoff     | logoff_user_email_key |   false    |            1 | user_email  | ASC       |  false  |  false   |
-| logoff     | logoff_user_email_key |   false    |            2 | user_id     | ASC       |  false  |   true   |
-+------------+-----------------------+------------+--------------+-------------+-----------+---------+----------+
-(3 rows)
+table_name |   index_name   | non_unique | seq_in_index | column_name | direction | storing | implicit
++------------+----------------+------------+--------------+-------------+-----------+---------+----------+
+users      | primary        |   false    |            1 | id          | ASC       |  false  |  false
+users      | users_name_key |   false    |            1 | name        | ASC       |  false  |  false
+users      | users_name_key |   false    |            2 | id          | ASC       |  false  |   true
+users      | users_dl_key   |   false    |            1 | dl          | ASC       |  false  |  false
+users      | users_dl_key   |   false    |            2 | id          | ASC       |  false  |   true
```

Comment:
This table appears to be a little out of alignment. #nitpicking

Reply:
Great catch!